### ISSUE TITLE
Hello, may I know how to train this neural network model?

### ISSUE THREAD

**Issue by cao3082423114** [kai422/CoVOS#2]
I have looked through the code, but I couldn't find how to train the network. Could you please tell me how to train the network?

**Comment by kai422** [comment]
Hi,

Sorry for the late response. The training is based on the code from https://github.com/hkchengrex/Mask-Propagation. But you will need to generate motion vectors and residuals in advance for training. I will release the training code later.

Regards,
Kai

**Comment by cao3082423114** [comment]
Thank you very much for taking the time to reply to me, and thank you for providing me with training methods. I am also looking forward to your code on generating motion vectors and residual graphs.
Best regards,
cao1828030626

**Comment by Sheldon04** [comment]
Hello! I'm very interested in your work! I wonder that when the training code will be released?
Best wishes 
Tian
